node/config.go

```diff
@@ -97,6 +98,44 @@ type Config struct {
 	// handshake phase, counted separately for inbound and outbound connections.
 	// Zero defaults to preset values.
 	MaxPendingPeers int
+
+	// HttpHost is the host interface on which to start the HTTP RPC server. If this
+	// field is empty, no HTTP API endpoint will be started.
+	HttpHost string
```

Comment:
Could you change this to HTTPHost (same for the other fields)?
We use this [convention](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) on most places.